rb/lib/selenium/webdriver/remote/response.rb

```diff
@@ -59,34 +56,12 @@ def assert_ok
           raise Error::ServerError, self
         end
 
-        def add_backtrace(ex, server_trace)
```

Comment:
I think this logic was probably still required, it's not this logic that should have been removed, but simply where the backtrace was being set.

Reply:
The following comment directs to the reasoning behind this